### ISSUE TITLE
Make a stats page

### ISSUE THREAD

**Issue by ahankinson** [diamm/diamm#57]
Make a page with statistics about DIAMM -- how many sources, images, etc.

**Comment by belljustin** [comment]
List of stats to include
- sources
- sources with images
- archives
- people<sup>1</sup>
- compositons

<sup>1</sup>are people exclusively composers?

**Comment by ahankinson** [comment]
No, people can be composers, but they can also be copyists, or related to a source, or all three.
